src/common/transformations/tests/op_conversions/convert_convertaligntypes_test.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_convertaligntypes.hpp"
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_align_types.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/pass/manager.hpp"
+
+using namespace ov;
+using namespace ov::op;
```

Comment:
Would be better to remove them and replace to pull what is required like parameter, PartialShape etc.